build.sbt

```diff
@@ -80,9 +81,12 @@ lazy val seleniumJSEnv: Project = project.
     name := "scalajs-env-selenium",
 
     libraryDependencies ++= Seq(
+        /* Make sure selenium is before scalajs-envs-test-kit:
```

Comment:
Nope, we need the dependency on closure-compiler, but it needs to be later in the classpath.